src/components/sections/ProfileForm.tsx

```diff
@@ -1,23 +1,42 @@
 import "../../styles/form.css";
 import Input from "../common/Input";
 import TechnologyForm from "./TechnologyForm";
-import { useState } from "react";
 import { DataType } from "../../types/DataType";
 import { FiX } from "react-icons/fi";
+import { useFormik } from "formik";
+import * as Yup from "yup";
 
 type Props = {
   save: (data: DataType) => void;
 };
 
+const validationSchema = Yup.object().shape({
+  fullName: Yup.string()
+    .min(3, "Name must be at least 2 characters")
+    .max(25, "Name must be a maximum of 25 characters")
+    .required("Name is required"),
+  position: Yup.string().required("Position is required"),
+  city: Yup.string()
+    .min(3, "City must be at least 2 characters")
```

Comment:
I think the message is wrong. Shouldn't be 3 characters?

Yup provides a function for the message, so you can get parameters from it. Can you try that in order to avoid rewriting the min value?
```suggestion
    .min(3, ({ min }) => `City must be at least ${min} characters`)

```